site/src/modules/resources/AgentMetadata.tsx

```diff
@@ -34,9 +34,11 @@ export const AgentMetadataView: FC<AgentMetadataViewProps> = ({ metadata }) => {
   }
   return (
     <section css={styles.root}>
-      {metadata.map((m) => (
-        <MetadataItem key={m.description.key} item={m} />
-      ))}
+      {[...metadata]
+        .sort((a, b) => a.order - b.order)
```

Comment:
We sort these items on the backend side. We don't need to sort it one more time here. If there is a bug, it should be fixed on the backend side.